library/ssl_tls13_generic.c

```diff
@@ -3116,7 +3116,8 @@ int mbedtls_ssl_write_early_data_ext( mbedtls_ssl_context *ssl,
 #if defined(MBEDTLS_SSL_CLI_C)
     if( ssl->conf->endpoint == MBEDTLS_SSL_IS_CLIENT )
     {
-        if( ssl->conf->key_exchange_modes == MBEDTLS_SSL_TLS13_KEY_EXCHANGE_MODE_ECDHE_ECDSA ||
+        if( !mbedtls_ssl_conf_tls13_some_psk_enabled( ssl ) ||
+            mbedtls_ssl_get_psk_to_offer( ssl, NULL, NULL, NULL, NULL ) != 0 ||
```

Comment:
Should this line be `mbedtls_ssl_get_psk_to_offer( ... )` to be in the same style as the line above?

Reply:
The return value of `mbedtls_ssl_get_psk_to_offer)()` is checked agains `0`, in other places. So let's follow that.